README.md

```diff
@@ -0,0 +1,75 @@
+# laravel-weather-apis
+
+## Run the app via Docker
+
+Build a Docker image for the app:
+
+Please ask Erika for the weather api key.
+```bash
+docker build --tag weather-app --build-arg token=<YOUR_AUTH_TOKEN> --build-arg key=<WEATHER_API_KEY> .
```

Comment:
Seeing this and thinking about what it means for a customer to run it (with the super long php grpc build time), I'm feeling motivated to have us tackle this ticket soon:

https://github.com/momentohq/dev-eco-issue-tracker/issues/131

That is not a blocker for this PR.  But if you would be interested in working on that after we get this merged in please let me know and we can bump it up the priority list a bit, and then use it to improve the user experience for this laravel app.